Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cid"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Please don't increase the version of package. This is done by the release process.

I'll push a new commit myself, I don't want to bug you more than needed :)

Reply:
It turns out I can't. So @sunny-g, please make this change yourself :)

Reply:
My bad, though I overreached :)